src/test/clojure/xtdb/operator/table_test.clj

```diff
@@ -112,7 +112,7 @@
                         {:params {'?x53 "RAIL"}}))))
 
 (t/deftest test-date-subtraction-bug-435
-  (t/is (= [{:a #time/duration "PT24H"}] ; it returned PT24000000H :/
```

Comment:
mind duplicating this test rather than updating it? was a very specific bug that I wanted to keep a regression test for

Reply:
What do we want to duplicate here, exactly? Would it be using timestamps or something like that?

Reply:
hmm, maybe it is date-specific, so the issue goes away if `date - date` becomes integer... :thinking: 

alright, let's kill it - these comments should link from #2424 so should spot it when we come back to it